Makefile

```diff
@@ -282,6 +286,15 @@ docker-build: ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	docker push $(IMG)
 
+kind-load-catalog: ## Load catalog image to local cluster
```

Comment:
no need to load the catalog with `kind load`. The catalogsource sets the image pull policy to `Always`, so even if the image is loaded in the kind cluster, it will fail if the image is not in quay. Right @didierofrivia ?

Reply:
True that, sadly there's no way to use local cluster images, so no need to push the catalog to kind